solidity/contracts/Chainlinked.sol

```diff
@@ -66,8 +85,10 @@ contract Chainlinked {
     link = LinkToken(_link);
   }
 
-  modifier onlyOracle() {
-    require(msg.sender == address(oracle));
+  modifier checkChainlinkRequest(bytes32 _requestId) {
```

Comment:
I love the idea. I'm just worried about using the word `Request` for both directions. i.e. We make a chainlink request from requester -> oracle -> node, and then here, we have `checkChainlinkRequest` that's from the node -> oracle -> fulfiller.

The word `request` is being used for both directions, and then at the end of it, we `emit ChainlinkFulfilled`.